presto-spi/src/main/java/com/facebook/presto/spi/block/AbstractRowBlock.java

```diff
@@ -13,6 +13,8 @@
  */
 package com.facebook.presto.spi.block;
 
+import com.facebook.presto.spi.memory.ArrayPool;
```

Comment:
Are these imports necessary? Did the file compile before?

Reply:
Sorry I forgot to remove it. I was trying to split the commit. My apologies.